### DISCUSSION TITLE
VoLTE and VoWiFi carrier Beeline Russia (in Montenegro)

### DISCUSSION THREAD

**Discussion by kuznetsovp-tipico** [kyujin-cho/pixel-volte-patch#97] [Carrier-specific Experiences]
Hello @kyujin-cho,

  Thank you so much for starting.

  Second, report it has been _working_ for carrier Beeline Russia in Montenegro country (in the roaming). 

  Pixel 6 Pro: **TQ1A.230205.002**
  Pixel IMS: **ver 1.2.4**
  Shizuku: **ver 13.2.1.t958.5f9516b**
  MMC: **250**
  MNC: **99**
  
  VoWIFI: **is working**
  VoLTE: **is not working.**
  
  **Nonetheless, when I tried to enable the option of Roaming preference the IMS registration is doesn't work.**
  
   I would like to ask you if you can please add the table to the compatibility matrix. "[Compatibility matrix for known carriers](https://github.com/kyujin-cho/pixel-volte-patch/blob/main/docs/compatibility-chart.en.md)" from here.

**Comment by kyujin-cho** [comment]
Thanks for the report! I've updated the compat matrix.

**Reply by kuznetsovp-tipico** [reply]
Hi @kyujin-cho,
updated after some tests and with additional information.

**Reply by kuznetsovp-tipico** [reply]
and BTW I am not sure that I got these symbols from the table - what do they mean?

**Reply by kyujin-cho** [reply]
What kind of symbols?

**Reply by qrilka** [reply]
I guess it's about O and X in the table
